src/app/card/info-status-card/info-status-card.component.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  Component,
+  Input,
+  OnInit
+} from '@angular/core';
+
+/**
+ * Info Status Card Component
+ */
+@Component({
+  selector: 'pfng-info-status-card',
+  templateUrl: './info-status-card.component.html',
+  styleUrls: ['./info-status-card.component.less']
+})
+
+export class InfoStatusCardComponent implements OnInit {
+
+  @Input('showTopBorder') showTopBorder: boolean;
+  @Input('status') status: object;
+  @Input('htmlContent') htmlContent: boolean;
+
+  constructor() {}
+
+  ngOnInit(): void {}
```

Comment:
Public and protected methods also need to be documented. You can view the docs by selecting the API tab in your example.